consensus/consensus.go

```diff
@@ -671,11 +671,11 @@ func (consensus *Consensus) accumulateRewards(
 		}
 		numAccounts += len(stakeInfos)
 		for _, stakeInfo := range stakeInfos {
-			utils.GetLogInstance().Info("accumulateRewards: rewarding",
-				"block", header.Hash().Hex(),
-				"account", stakeInfo.Address.Hex(),
-				"node", hex.EncodeToString(stakeInfo.BlsAddress[:]),
-				"amount", BlockReward)
+			//			utils.GetLogInstance().Info("accumulateRewards: rewarding",
```

Comment:
This needs to stay.  Node operators will rely on this log line and wallet to monitor their reward payouts.